ee/clickhouse/views/test/test_experiment_saved_metrics.py

```diff
@@ -212,6 +215,51 @@ def test_create_update_experiment_saved_metrics(self) -> None:
         self.assertEqual(Experiment.objects.get(pk=exp_id).saved_metrics.count(), 0)
         self.assertEqual(ExperimentToSavedMetric.objects.filter(experiment_id=exp_id).count(), 0)
 
+    def test_create_saved_metric_with_experiment_metric(self):
+        response = self.client.post(
+            f"/api/projects/{self.team.id}/experiment_saved_metrics/",
+            data={
+                "name": "Test Experiment saved metric",
+                "description": "Test description",
+                "query": {
```

Comment:
It would be nice to rename this field now to `metric` to be consistent with new naming. That will require a db migration though, so maybe do it later as a separate isolated task?

Reply:
We don't get nice things here 😄 https://posthog.com/handbook/engineering/databases/schema-changes#do-not-delete-or-rename-django-models-and-fields

If we wanted to be exact about it, we'd introduce a new `metric` column. However, I don't think it's worth the complexity/effort/etc.